HACKING.md

```diff
@@ -269,17 +269,27 @@ Recommended tone:
 
 ## Creating a release
 
-To create a new tagged release, go to the [GitHub Releases page](https://github.com/canonical/pebble/releases) and:
+When releasing, you need to release the `master` branch and the `fips` branch.
 
-- Update `Version` in `cmd/version.go` to the version you're about to publish, for example `v1.9.0`, open a pull request, have it reviewed and merged into the `master` branch.
-- Open a pull request to merge the `master` branch into the `fips` branch, resolve any conflicts, and have it reviewed and merged.
-- Click ["Draft a new release"](https://github.com/canonical/pebble/releases/new).
-- Enter the version tag (for example `v1.9.0`) and select "Create new tag: on publish".
+Binaries will be created and uploaded automatically to this release by the [binaries.yml](https://github.com/canonical/pebble/blob/master/.github/workflows/binaries.yml) GitHub Action. In addition, a new Snap version is built and uploaded to the [Snap Store](https://snapcraft.io/pebble) (but promotion to `stable` is a manual step).
+
+Follow these steps:
+
+### Main release (master branch)
+
+- Update `Version` in `cmd/version.go` to the version you're about to publish, for example `v1.27.0`, open a pull request, have it reviewed and merged into the `master` branch.
+- [Draft a new GitHub release](https://github.com/canonical/pebble/releases/new).
+- Enter the version tag (for example `v1.27.0`) and select "Create new tag: on publish".
 - Enter a release title: include the version tag and a short summary of the release.
 - Write release notes: describe new features and bug fixes, and include a link to the full list of commits.
 - Click "Publish release".
-- Likewise, publish a FIPS release (for example `v1.9.0-fips`) targeting the tip of the `fips` branch.
-- Once the release GitHub Actions have finished, and the new [Snap](https://snapcraft.io/pebble) has been successfully built, update `Version` again to `v1.{next}.0-dev` (for example `v1.10.0-dev`).
```

Comment:
Oh, I removed this step: 1) because we forgot to do it half the time anyway, and 2) it doesn't seem to be important or particularly useful, as people always use the versioned releases anyway.